includes/admin/helper/class-wc-helper.php

```diff
@@ -665,6 +665,11 @@ private static function _helper_auth_connect() {
 			wp_die( 'Something went wrong' );
 		}
 
+		/**
+		 * Fires when the Helper connection process is initiated.
+		 */
+		do_action( 'woocommerce_helper_connect' );
```

Comment:
Probably will makes more sense to change to `woocommerce_helper_connect_init` like WP, or even `woocommerce_helper_connect_start`.

Reply:
Fixed in rebase on 612695f.